documentation/optee_design.md

```diff
@@ -523,8 +523,117 @@ implementation, as explained in [crypto.md].
 TBD
 
 # 12. Trusted Applications
-## Format
-Trusted Applications consists of a signed ELF file.
+
+## Static and Dynamic TAs
+
+There are two ways to implement Trusted Applications, static and dynamic.
```

Comment:
I would really prefer
 `There are two ways to implement secure services in OP-TEE: Trusted Applications or OP-TEE pseudo-TAs (also referred as "static TAs"). Trusted Applications (TAs) comply with GP specifications and are loaded at runtime. OP-TEE pseudo-TAs are OP-TEE core services using the GP TEE open/invoke/close interfaces as API.`